index.html

```diff
@@ -163,67 +189,107 @@ <h2 class="tonic4">Multi-Post Stories</h2>
+					<p class="am">I’m a software developer! I can help you build a product , feature or website Look
+						through some of my
+						work and experience! If you like what you see and have a project you need coded, don’t hestiate
+						to
+						contact me.</p>
+					<button type="button" class="connect">LET'S CONNECT</button>
+					<ul class="icons">
+						<li class="twitter"><img src="images/github.svg" alt="github svg"></li>
+						<li class="vector1"><img src="images/vector.svg" alt="vector"></li>
+						<li class="vector2"><img src="images/vectorrr.svg" alt="vector svg"></li>
+						<li class="medium"><img src="images/medium.svg" alt="medium svg"></li>
+						<li class="github"><img src="images/twitter.svg" alt="twitter svg"></li>
+						<li class="angelist"><img src="images/angellist.svg" alt="angellist"></li>
+					</ul>
+					<button class="resume">Get My Resume</button>
+				</div>
+				<div class="expertise">
+					<ul class="language">
+						<li>
+							<h3 class="mylanguage">Languages</h3>
+						</li>
+						<li><i class="fa fa-chevron-down" aria-hidden="true"></i></li>
+					</ul>
+					<div class="desk-programming">
+						<ul class="outer-item">
+							<li class="javascript"><img src="images/Ellipses.svg" alt="JavaScript"></li>
+							<li>
+								<p>JavaScript</p>
+							</li>
+						</ul>
+						<ul class="outer-item">
+							<li class="CSS"><img src="images/ELLIPSE.svg" alt="HTML"></li>
+							<li>
+								<p>CSS</p>
+							</li>
+						</ul>
+						<ul class="outer-item">
+							<li class="HTML"><img src="images/Ellipse.svg" alt=" CSS"></li>
+							<li>
+								<p>HTML</p>
+							</li>
+						</ul>
+					</div>
+					<ul class="framework">
+						<li class="flex-items">
+							<h3>Frameworks</h3>
+						</li>
+						<li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+					</ul>
+					<ul class="skills">
+						<li class="flex-items">
+							<h3 class="skills">Skills</h3>
+						</li>
+						<li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+					</ul>
+				</div>
+			</section>
+		</div>
+
+      <footer class="footer">
+
+		<div class="contact">
+			<h2 class="infor">Contact me</h2>
+			<p class="about-infor"> If you have an application you <br>
+				are interested in developing, a <br>
+				feature that you need built or a  <br>
+				project that needs coding.I’d <br>
+				 love to help with it </p>
+			<form action="handling-form" method="post">
+				<ul>
+				 <li>
+				   <label for="name"></label>
+				   <input class="name" type="text" placeholder="Enter your Name" id="name" name="user_name">
+				 </li>
+				 <li>
+				   <label for="mail"></label>
+				   <input class="email" type="email" placeholder="Enter your Email" id="mail" name="user_email">
+				 </li>
+				 <li>
+				   <label for="msg"></label>
+				   <textarea class="message" id="msg" name="user_message" placeholder="Write your text in here" rows="8" cols="30" maxlength="500"></textarea>
+				 </li>
```

Comment:
- [ ] I suggest adding `required` attribute in all the inputs and the text-area to activate the validation on submit ( to prevent the user from submitting an empty field )👌

![image](https://user-images.githubusercontent.com/45231276/186375922-cab19238-2ba0-4ccb-b155-44b18c2165d1.png)